apps/api/src/queries/photograph/__snapshots__/photograph-queries-fetch-by-id.e2e.spec.ts.snap

```diff
@@ -0,0 +1,198 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`when querying for a photograph: fetch by Id when the user is authenticated when the user is a coscrad admin when the user is a project admin when there is a photograph with the given Id when the photograph is published should return the expected result 1`] = `
+[
```

Comment:
Did we snapshot just the actions? Because that's what this is.

Reply:
I have fixed this. We need to snapshot the entire response body once and only once per each resource `fetchById` and `fetchMany` as a contract test. If the snapshot changes, we have introduced potential breaking changes to the client. We do this in one of the admin test cases (project admin by convention) so that we snapshot the actions as well.